Advance-1/api.php

```diff
@@ -0,0 +1,114 @@
+<?php 
+require '../vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+/**
+ * Class for extracting data from the API.
+ */
+class API
+{
+  // Array to store the body links of each section.
+  public $link_arr = array();
+
+  // Array to store the heading of each section.
+  public $heading_arr = array();
+
+  // Array to store the image of each section.
+  public $img_arr =  array();
+
+  // Array to store the redirect links.
+  public $redirect_link = array();
+
+  /**
+   * This Function accepts a url and return the data fetched from the api link in json format.
+   * @param [$url] -> API Link sent by the user.
+   * @return Array.
+   */
+  function get_json($url) {
+    $client = new Client([
+      'base_uri' => $url,
+    ]);
+    $response = $client->request('GET', '',);
+    $body = $response->getBody();
+    $arr_body = json_decode($body);
+    return $arr_body;
+  }
+
+  /**
+   * This function returns the body data of each section along with its link. 
+   * @param [$data] -> JSON data sent by the user which needs to be searched for the links.
+   * @return Array.
+   */
+  function get_services_link($data) {
+    $i = 0;
+    foreach($data->data as $apidata) {
+      if (!($apidata->attributes->field_services) == null) {
+        $link_arr[$i] = $apidata->attributes->field_services->value;
+        $i++;
+      }
+    }
+    return $link_arr;
+  }
+
+  /**
+   * This function returns the heading data of each section. 
+   * @param [$data] -> JSON data sent by the user which needs to be searched for the heading.
+   * @return Array.
+   */
+  function get_services_headings($data) {
```

Comment:
Please put camel case for naming a function.

Reply:
https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-functions-and-variables